utilities/common.py

```diff
@@ -130,10 +130,7 @@ def common_parser(for_reco_highPU=False):
     common_logger = logging.setup_logger(__file__, initargs.verbose, initargs.noColorLogger, initName="common_logger_wremnants")
     
     import ROOT
-    if not initargs.nThreads:
-        ROOT.ROOT.EnableImplicitMT()
-    elif initargs.nThreads != 1:
-        ROOT.ROOT.EnableImplicitMT(initargs.nThreads)
+    ROOT.ROOT.EnableImplicitMT(initargs.nThreads)
```

Comment:
So -1 is a valid argument here?

Reply:
No, negative values must not be allowed, but 1 is valid, it means one thread. I'll add a protection against negative values, or we could just transform it to 0 (if a user insists in using invalid values they will get what I decide for them :) )